src/runtime.cpp

```diff
@@ -163,8 +157,12 @@ static int rt_curl_debug_callback(CURL* handle, curl_infotype type, char* data,
 }
 #endif
 
-runtime::runtime(std::string const& endpoint)
-    : m_endpoints{{endpoint + "/2018-06-01/runtime/init/error",
+runtime::runtime(std::string const& endpoint) : runtime(endpoint, "AWS_Lambda_Cpp/" + std::string(get_version()))
+{}
+
+runtime::runtime(std::string const& endpoint, std::string const& user_agent)
+    : m_user_agent_header("User-Agent: " + user_agent + ""),
```

Comment:
What's the last empty string for?

Reply:
nothing, I did a bad job cleaning up from a previous revision